polymath/extensions/pickler.py

```
@@ -90,920 +90,928 @@
 
-################################################################################
-# Support for fpzip compression and decompression
-################################################################################
+        try:
+            references = references[:2]
+            for reference in references[:2]:
+                if isinstance(reference, numbers.Real):
+                    pass
+                elif reference not in {'smallest', 'largest', 'mean', 'median', 'logmean',
+                                       'fpzip'}:
+                    raise ValueError('invalid pickle reference {reference!r}')
```

Ruff (0.14.7)

Abstract `raise` to an inner function

(TRY301)